examples/sites/src/views/components/components/anchor.vue

```
@@ -0,0 +1,116 @@
+      :offset-top="56"
+      :links="links"
+      :is-affix="props.anchorAffix"
+      type="dot"
+      container-id="#doc-layout-scroller"
+      @link-click="handleAnchorClick"
+    >
+    </tiny-anchor>
+  </div>
+</template>
+<script setup lang="ts">
+import { ref, computed } from 'vue'
```

ESLint

[error] 'ref' is defined but never used.

(unused-imports/no-unused-imports)